### ISSUE TITLE
empty or missing text_file causes dropped notification

### ISSUE THREAD

**Issue by pivotal-dylan-roberts** [cloudfoundry-community/slack-notification-resource#46]
steps to reproduce:
 - clone https://github.com/pivotal-dylan-roberts/slack-file-issue (or just copy the pipeline.yml from the repo)
 - set the pipeline defined by pipeline.yml, including a valid slack web hook url and channel name as variables
 - manually trigger each of the included jobs in the pipeline

expected: each job produces a slack notification in the specified channel
actual: no notifications are produced

**Comment by matthope** [comment]
Looks like the "simplest" fix for this is:

```
--- a/out
+++ b/out
@@ -53,6 +53,7 @@ then
 fi

 export TEXT_FILE_CONTENT=""
+[[ -n "${text_file}" && ! -f "${text_file}" ]] && text_file=""
 [[ -n "${text_file}" && -f "${text_file}" ]] && TEXT_FILE_CONTENT="$(cat "${text_file}")"

 ATTACHMENTS_FILE_CONTENT=""
```

There is a check to see if the file exists before setting "TEXT_FILE_CONTENT", but later checks are just if the value is set, not if it exists. 

In my testing, this addresses both example jobs provided.

**Comment by jhunt** [comment]
Merged in #47
